src/components/BlockEditor/index.tsx

```
@@ -0,0 +1,72 @@
+					const edJsData = await api.saver.save();
+					const htmlArr = edjsParser.parse(edJsData);
+					onChange?.(htmlArr.join(''));
+				},
+				placeholder: 'Start writing your content here...'
+			});
+			ref.current = editor;
+		}
+
+		//Add a return function to handle cleanup
+		return () => {
+			if (ref.current && ref.current.destroy) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)